src/python/cli_new/lib/cli/mesos.py

```diff
@@ -539,6 +554,16 @@ def _launch_nested_container_session(self):
         nested container and attach to its output stream.
         The output stream is then sent back in the response.
         """
+
+        # Set authentication header
+        # pylint: disable=line-too-long
+        auth = None
+        if self.config.agent_principal() is not None and self.config.agent_secret() is not None:
```

Comment:
You can just wrap the line instead of suppressing pylint warnings:
```
if self.config.agent_principal() is not None and \
    self.config.agent_secret() is not None:
```